app/style/themes/playful.css

```diff
@@ -0,0 +1,777 @@
+/*
+ * Theme: Playful
+ * Author: Luis Fernando Saavedra Meza <saavedra.f12@gmail.com>
+ */
+
+/*
+ * ==========================================================================
+ * Font Importing
+ * ==========================================================================
+ */
+@import url('https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700');
+
+
+/*
+ * ==========================================================================
+ * Common Elements Styling
+ * ==========================================================================
+ */
+
+ ::-webkit-scrollbar {
```

Comment:
Scroll bar properties will overwrite any other styling since they can't be set per class, these settings do not disrupt any other so they should be added in the main.css file instead. The properties in that file are right here: https://github.com/Skrifa/Skrifa/blob/master/app/style/main.css#L80